linux/raidcontroller.go

```diff
@@ -0,0 +1,16 @@
+package linux
+
+import "machinerun.io/disko"
+
+type RAIDControllerType string
+
+const (
+	MegaRAIDControllerType RAIDControllerType = "megaraid"
+)
+
+type RAIDController interface {
+	// Type() RAIDControllerType
```

Comment:
Why comment this out?  A Type() is probably quite useful.  Not sure if it should just return string instead of RAIDControllerType, probably not...

Reply:
Ah,  I remember.  I needed to avoid import loops.  If I want to specify a RAIDControllerType under megaraid or smartpqi, then I have to import linux package, but linux package imports the megaraid/smartpqi package.

Reply:
Ah - but then having it return a string would avoid that, right?

Reply:
It's not a big deal, btw - we can add it when someone needs it.